Content/1.1.4 - Interaction Review/interactionReview.json

```diff
@@ -0,0 +1,3 @@
+{
+	"interactionReview_possibleDappCalls": "Possible dApp calls"
```

Comment:
There are lot of UI components that are going to be used in both `Transaction Review` & `Pre-Authorization Review`. Such components are already implemented using strings defined in [transactionReview.json](https://github.com/radixdlt/apps-localization-src/blob/main/Content/1.1%20-%20Transaction%20Review/transactionReview.json#L21). 

Such strings don't belong only to `Transaction`, as they are also part of `PreAuthorization`. Therefore, it makes more sense to have them moved into a new common `Interaction` file that groups both of them. 

I am happy to move them here, but would need approval from other team members (specially Android team) before doing so. 

For now, just adding this new Strings here. Every new String that is required to extend these components should be placed here as well.

Reply:
I agree. I can make the change on android if you move the common strings.

Reply:
that would be great, thanks! 

I would recommend first just copying all the strings here, and then once they have been migrated on host apps (and merged), we delete them from `transaction_review.XYZ`. Wdyt?

Reply:
Sounds good